packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx

```diff
@@ -1,6 +1,339 @@
+  StreamMeeting,
+  StreamVideo,
+  ToggleAudioPublishingButton,
+  ToggleCameraPublishingButton,
+} from '@stream-io/video-react-sdk';
+
+// ... API key, provider, and user setup
+
+export const HostApp = () => {
+  const client = useCreateStreamVideoClient({
+    apiKey,
+    tokenOrProvider,
+    user,
+  });
+
+  return (
+    <StreamVideo client={client}>
+      <StreamMeeting
+        callType="livestream"
+        callId="some-random-call-id"
+        autoJoin={true}
+        data={{
+          create: true,
+          // optionally, invite other hosts to the live stream
+          data: {
+            members: [...],
+          }
+        }}
+      >
+        <BackstageUI />
+      </StreamMeeting>
+    </StreamVideo>
+  )
+}
+
+const BackstageUI = () => {
+  const call = useCall();
+  const callState = useCallCallingState();
+  // useMediaDevices() hook provides you quick access to the Browser's MediaDevices API
+  // and a few helper methods to manage your audio and video streams.
+  const { publishVideoStream, publishAudioStream } = useMediaDevices();
+  useEffect(() => {
+    if (!call) return;
+    if (callState === CallingState.JOINED) {
+      // once the call is joined, publish the host's audio and video
+      publishVideoStream();
+      publishAudioStream();
+    }
+  }, [call, callState, publishAudioStream, publishVideoStream]);
+  return (
+    <>
+      <BackstageHeader />
+      // use a predefined layout from the SDK or build your own
+      <PaginatedGridLayout />
+      <BackstageControls />
+    </>
+  );
+};
+
+const BackstageControls = () => {
+  return (
+    <div className="backstage-controls">
+      <ToggleAudioPublishingButton />
+      <ToggleCameraPublishingButton />
+      <CancelCallButton />
+      // The implementation of this button is shown in the next step
+      <ToggleLivestreamButton />
+    </div>
+  );
+};
+
+```
+
+With this done, we have a fully functional livestream experience for the host.
+The hosts can join the live stream, publish their audio and video, and invite other hosts to the live stream.
+
+:::note
+When the call is started, the live streaming isn't enabled for the viewers yet. 
+We will see how to enable it in the next step.
+::: 
+
+
+## Step 3 - Enable the live stream for the viewers
+
+For simplicity, we are going to implement an additional button in the `BackstageControls` component that will enable the live stream for the viewers.
+
+The button should be able to toggle the live stream on and off. For that purpose, we are going to use two utility hooks from the SDK: 
+- [`useCall`](../04-call-engine/hooks-and-contexts.md#usecall) and
+- [`useIsCallBroadcastingInProgress`](../04-call-engine/hooks-and-contexts.md#useiscallbroadcastinginprogress)
+
+
+The `useCall` hook provides you with the current call object. This is useful when you want to access the call object from a component that is deep in the component tree with the `<StreamMeeting />` component as a predecessor.
```

Comment:
Let's keep it as "we" all the way through, switching between it and "you" is a bit confusing I think.

```suggestion
The `useCall` hook provides us with the current call object. This is useful when we want to access the call object from a component that is deep in the component tree with the `<StreamMeeting />` component as a predecessor.
```